packages/fxa-settings/src/pages/Pair/Index/en.ftl

```diff
@@ -0,0 +1,18 @@
+## Pair index page
+
+pair-sync-header = Sync { -brand-firefox } on your phone or tablet
+pair-cad-header = Connect { -brand-firefox } on another device
+pair-already-have-firefox-paragraph = Already have { -brand-firefox } on a phone or tablet?
+# Clicking this button initiates the pairing process, usually by directing the user to the `about:preferences` page in Firefox
+pair-sync-your-device-button = Sync your device
+# This is an h2 element immediately preceded by "Sync your device"
+pair-or-download-subheader = Or download
+# Directs user to scan a QR code to download Firefox. <linkExternal> is an anchor tag that directs the user to where they can download the { -brand-firefox } app
+pair-scan-to-download-message = Scan to download { -brand-firefox } for mobile, or send yourself a <linkExternal>download link.</linkExternal>
```

Comment:
I know, I know, it's a nit 😅 , but the period should be outside the link in the strings and in the fallback text.

Reply:
+1, also I feel like a lot of my comments end up being nits but it does improve the final quality of the localized versions.

Reply:
Nits are welcome!!! I'll fix it, sorry I misunderstood that one!